apps/generic-issuance-client/src/pages/pipeline/PipelineEditSection/PipelineEditSection.tsx

```diff
@@ -23,165 +27,104 @@ export function PipelineEditSection({
-          isAdminView={false}
-          singleRowMode={true}
-        />
+    <Container>
+      <Box padding={0} mb={0} flexShrink={0}>
+        <SinglePipelineTable {...{ pipeline, pipelineInfo }} />
       </Box>
 
-      <FancyEditor
-        style={{ width: "800px", height: "450px" }}
-        language="json"
-        value={editorValue}
-        setValue={setEditorValue}
-        readonly={ownedBySomeoneElse && !isAdminView}
-      />
-
-      {isAdminView || !ownedBySomeoneElse ? (
-        <HStack minWidth="fit-content">
-          {hasEdits && (
-            <Button
-              variant="outline"
-              size="sm"
-              isDisabled={
-                !!actionInProgress || (ownedBySomeoneElse && !isAdminView)
-              }
-              onClick={onSaveClick}
-            >
-              {actionInProgress ? "Saving..." : "Save Changes"}
-            </Button>
-          )}
-          {!hasEdits && (
-            <Button size="sm" isDisabled={true} variant="outline">
-              Save Changes
-            </Button>
-          )}
-
-          <Button
-            variant="outline"
-            onClick={onUndoClick}
-            size="sm"
-            isDisabled={!hasEdits}
-          >
-            Reset Changes
-          </Button>
+      <div style={{ flexGrow: 1, flexShrink: 1, overflow: "hidden" }}>
+        <Maximizer
+          maximized={editorMaximized}
+          setMaximized={setEditorMaximized}
+        >
+          <FancyEditor
+            dark
+            value={editorValue}
+            setValue={setEditorValue}
+            readonly={(ownedBySomeoneElse && !isAdminView) || !!historyEntry}
+            ref={editorRef}
+            editorStyle={{
+              width: editorMaximized ? "100%" : "100%",
+              height: editorMaximized ? "100vh" : "100%"
+            }}
+            containerStyle={
+              editorMaximized ? { border: "none", borderRadius: 0 } : undefined
+            }
+            editorOptions={
+              editorMaximized
+                ? {
+                    minimap: {
+                      enabled: true
+                    }
+                  }
+                : undefined
+            }
+            language="json"
+          />
+        </Maximizer>
+      </div>
 
-          <Button
-            variant="outline"
-            size="sm"
-            colorScheme="red"
-            isDisabled={ownedBySomeoneElse && !isAdminView}
-            onClick={onDeleteClick}
-          >
-            Delete Pipeline
-          </Button>
-
-          {isAdminView && (
-            <Button
-              variant="outline"
-              size="sm"
-              isDisabled={ownedBySomeoneElse && !isAdminView}
-              onClick={onDuplicateClick}
-            >
-              Duplicate Pipeline
-            </Button>
-          )}
-        </HStack>
-      ) : (
-        <></>
-      )}
-    </Stack>
+      <div
+        style={{
+          flexShrink: undefined
```

Comment:
What does this do? I would naively assume that it has no effect.

Reply:
changed this to equal `"1"` rather than `undefined`, which allows the edit section to shrink in case the page size changes